.npmignore

```diff
@@ -1 +1,3 @@
-sample/
+example
+.github
+scripts
```

Comment:
is it possible this is causing the build failure you're currently seeing. At first glance it looks as though it can't find the ci script.

Reply:
The build script use to be in the samples directory for whatever reason. I moved it to be a top level folder.